bigtable/integration_test.go

```diff
@@ -2128,6 +2148,31 @@ func TestIntegration_AdminBackup(t *testing.T) {
 	if _, err := adminClient.TableInfo(ctx, restoredTable); err != nil {
 		t.Fatalf("Restored TableInfo: %v", err)
 	}
+	// Restore backup to different instance
+	diffTable := table + "-diff-restored"
+	diffConf := IntegrationTestConfig{
+		Project:  testEnv.Config().Project,
+		Instance: diffInstance,
+		Cluster:  diffCluster,
+		Table:    diffTable,
+	}
+	env := &ProdEnv{
+		config: diffConf,
+	}
+	dAdminClient, err := env.NewAdminClient()
```

Comment:
Looks like you already have an admin client initiated upper

Reply:
The admin client initiated upper is the client connected to the source instance and the source table. The new `dAdminClient` has connection with the different instance `diffInstance` and the new restored table `diffTable`.

Reply:
Okay. Is it working? Overall LGTM

Reply:
Maybe there's a way how to check restored table on the different instance without creating a new admin client, but I couldn't find it.

Reply:
Unfortunately, it still failing for me with the error:
```
--- FAIL: TestIntegration_AdminBackup (26.02s)
    /home/sasha/go/src/google-cloud-go/bigtable/integration_test.go:2170: RestoreTableTo: rpc error: code = InvalidArgument desc = Request contains an invalid argument.
```

Reply:
Aren't you missing a database id in the table/backup paths? I think it should be `/instances/instance-id/database/database-id/tables/table-id`

Reply:
The problem was with instance zone, we had an email from Bonan where he proposed to change instance zone to `us-east1 `or similar. Alex and me tested and it works for both of us. So I pushed commit with last changes.